node-calendar/node-calendar.d.ts

```diff
@@ -0,0 +1,271 @@
+// Type definitions for node-calendar v0.1.4
+// Project: https://www.npmjs.com/package/node-calendar
+// Definitions by: Luzian Zagadinow <https://github.com/luzianz>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IDayOfWeekMonth extends Array<number> {

```

Comment:
I propose this base type would use tuple type(e.g. [number, number]) instead of number of array.